app/views/concerns/facet_presenter.rb

```diff
@@ -0,0 +1,124 @@
+module FacetPresenter
+  extend ActiveSupport::Concern
+
+  protected
+
+  def facet_item_url(facet, item)
+    if facet_in_params?(facet, item)
+      search_action_url(remove_facet_params(facet, item, params))
+    else
+      search_action_url(add_facet_params_and_redirect(facet, item))
+    end
+  end
+
+  def facet_display(facet, children = false)
```

Comment:
Unused method argument - `children`. If it's necessary, use `_` or `_children` as an argument name to indicate that it won't be used.